boards/particle-argon/doc.txt

```diff
@@ -0,0 +1,24 @@
+/**
+@defgroup    boards_particle-argon Particle Argon
+@ingroup     boards
+@brief       Support for the Particle Argon
+
+### General information
+
+[Particle Argon](https://docs.particle.io/argon/) is a mesh-ready development kit
+that provides access to multiple communication protocols: BLE, 802.15.4 and WiFi.
+
+img src="https://docs.particle.io/assets/images/argon/argon-top.png"
```

Comment:
This is not showing right when displaying documentation.

Reply:
Fixed